bitnami/keycloak/values.yaml

```
@@ -251,7 +251,10 @@ containerPorts:
   https: 8443
 ## @param extraContainerPorts Optionally specify extra list of additional port-mappings for Keycloak container
 ##
-extraContainerPorts: []
+extraContainerPorts: 
+  - name: infinispan
+    containerPort: 7800
+    protocol: TCP
```

This is not necessary to change. 
```suggestion
extraContainerPorts: []
```
You must add this part to the container port:
```yaml
containerPorts:
  http: 8080
  https: 8443
  infinispan: 7800
``` 
And in the satefulset.yaml:
```yaml
   ...
          ports:
          ...
            - name: infinispan
              containerPort: {{ .Values.containerPorts.infinispan }}
              protocol: TCP
...
```